docs/07-deployments/04-deploying-to-aws.md

```diff
@@ -118,6 +132,27 @@ _Connect to the database with Postico._
 
 ## Deploy your code
 
+:::caution
+
+using an old version of serverpod cli will generate the github action file containing old dart version. You can make the following fixes:
+
+1. in `./github/workflows/deployment-aws.yml` use
+
+```yaml
+      - name: Setup Dart SDK
+        uses: dart-lang/setup-dart@v1.6.5
+        with:
+          sdk: 3.5
+```
+
+1. in `mypod_server/config/production.yaml` change the version of installed dart like example
+
+```bash
+wget -q https://storage.googleapis.com/dart-archive/channels/stable/release/3.5.1/sdk/dartsdk-linux-x64-release.zip
+```
```

Comment:
Suggestion: Add a description on how to find the latest dart version or the list of all these URLs. Perhaps how to modify the version. That is not completely obvious right now.